tools/pot/tests/test_scales.py

```diff
@@ -163,8 +166,8 @@ def test_weights_scales(tmp_path, models):
         ref_min, ref_max = ref_weights[fq_name]['low_level'], ref_weights[fq_name]['high_level']
         assert_flag = False
 
-        if np.array_equal(ref_min, item_min) and \
-                np.array_equal(ref_max, item_max):
+        if np.allclose(ref_min, item_min, atol=EPS) and \
+                np.allclose(ref_max, item_max, atol=EPS):
```

Comment:
Can we ignore the relative tolerance `rtol` which is set to `1e-5`?
https://numpy.org/doc/stable/reference/generated/numpy.allclose.html

Reply:
Done, please check